Google.PowerShell/Logging/GcLogCmdlet.cs

```diff
@@ -640,4 +640,53 @@ protected override void ProcessRecord()
             WriteLogEntriesResponse response = request.Execute();
         }
     }
+
+    /// <summary>
+    /// <para type="synopsis">
+    /// Remove logs.
+    /// </para>
+    /// <para type="description">
+    /// Removes a log by its name
```

Comment:
Use a complete sentence, i.e. end in a period.

Could you elaborate on this though? Does each log _entry_ have a name? Or are logs grouped by a name? e.g. "Removes log entries by the group name. For example, all logs associated with GCE VM "vm-alpha1" in project "gootoso" will be in a group named "gootoso/logs/vm-alpha1".